doc/source/guide/geometry_guide.rst

```diff
@@ -0,0 +1,286 @@
+.. _geometry_guide:
+
+##################################
+Tomographic acquisition geometries
+##################################
+
+
+This document is intended to introduce definitions and conventions used in ODL `Geometry` classes that represent tomographic acquisition geometries.
+The first part introduces the abstract description of data as a function on a data manifold and then shows how this representation is simplified by using a parametrization of the manifold with a tuple of real numbers.
+The second part then describes the implementation of these concepts in ODL.
+
+
+Mathematical introduction
+=========================
+Acquisition geometries are a central part of the ``odl.tomo`` subpackage.
+They specify the physical setup of tomographic experiments and thus encode the geometric information that is needed to relate measurement data to the precise configuration of the system that gave rise to this data.
+This geometric configuration defines the relation between a (usually unknown) spatial distribution of a physical quantity, e.g., an attenuation coefficient, to measured data, e.g., how many photons were counted per pixel for given source and detector positions in a tomographic scanner.
+
+
+Geometry and data structure
+---------------------------
+Mathematically, interaction between probing rays and physical matter is often modeled as integration along straight lines.
+The corresponding mathematical operation, called **ray transform**, incorporates all geometry information since it is usually defined as a mapping from a space of functions on :math:`\mathbb{R}^d` to a space of functions on a `manifold`_ :math:`M`, the *data manifold*.
+This data manifold is typically a subset of the manifold of all lines in :math:`\mathbb{R}^d`, and the value of a function in a certain point on that manifold corresponds to the value of the integral along that line.
+
+For instance, in 2 dimensions and parallel beam geometry, i.e., a setup where all lines from a common direction are parallel, the ray transform can be defined as
+
+    .. math::
+        &\mathcal{R} : L^2(\Omega) \to L^2(M),
+
+        &\mathcal{R}(f)(\theta, v) := \int_{\mathbb{R}} f(v + t\theta)\, \mathrm{d}t,\quad \theta \in \Gamma \subset \mathbb{S}^1,\ v \in \theta^\perp,
+
+where :math:`\Omega \subset \mathbb{R}^2` is a bounded domain, :math:`\mathbb{S}^1` is the unit sphere in 2 dimensions and :math:`\theta^\perp = \{x \in \mathbb{R}^2\, |\, \langle x,\, \theta \rangle = 0\}` is the plane (=line) perpendicular to a vector :math:`\theta`.
+In this case, the data manifold is
+
+    .. math::
+        M = \big\{(\theta, v)\,\big|\, \theta \in \Gamma,\ v \in \theta^\perp \big\}
+
+and encodes the subset of lines in :math:`\mathbb{R}^2` that are parallel to a unit vector in :math:`\Gamma`.
```

Comment:
No, it's a subset since we restrict `theta` to `Gamma`. There is no reference to `Omega` whatsoever at this point.